src/bin/smooth_scale.rs

```diff
@@ -280,10 +279,10 @@ fn main() {
     for (i, rul) in rule_1_set.iter().enumerate() {
         smooth(rul, i, &original, &mut scaled);
     }
-    for (i, rul) in rule_2_set.iter().enumerate() {
+    for (_i, _rul) in rule_2_set.iter().enumerate() {
```

Comment:
Commenting the for loop might be a better idea than renaming the vars

Reply:
done for both loops